tests/tests.sql

```diff
@@ -0,0 +1,147 @@
+-------------------------------
+-- 1. Create records 
+-------------------------------
+-- create crash ID 1, road type is `alley`
+insert into db.crashes_cris (crash_id, road_type_id)
+values (1, 1);
+-- create two units: vehicle and truck
+insert into db.units_cris (crash_id, unit_nbr, unit_type_id)
+values (1, 1, 1);
+insert into db.units_cris (crash_id, unit_nbr, unit_type_id)
```

Comment:
the `units_vz` records that were created here have null for `crash_id`, `crash_id` should not be nullable and should be carried over from the insert trigger on `units_cris` right?

Reply:
i am not sure! my thinking is that a VZ user could edit a unit's crash ID by populating `crash_id` on this table, but otherwise it should be `null` like any other column. otherwise we're creating redundant data when the `crash_id` is available through the `units_vz` -> `units_cris` -> `crashes_cris` relationship 🤔